config/locales/exceptions.en.yml

```diff
@@ -250,6 +250,12 @@ en:
           detail: "The %{size} items provided did not match the definition"
           code: 147
           status: 422
+        format:
```

Comment:
So I think keeping the same `title` and `detail` as `pattern` could lead to some confusion, since the only difference in the response is the `code` and `meta`.  I think `format` should mention that it's a built-in part of the spec and not a custom regex like `pattern` is.

That or we just combine them both into the `pattern` error. I'm fine with either.